salt/modules/nxos.py

```diff
@@ -332,9 +332,11 @@ def sendline(command, method='cli_show_ascii', **kwargs):
     '''
     smethods = ['cli_show_ascii', 'cli_show', 'cli_conf']
     if method not in smethods:
-        msg = "INPUT ERROR: Second argument 'method' must be one of {}".format(smethods)
-        msg = msg + '\n  Value passed: {}'.format(method)
-        msg = msg + '\n  Hint: White space separated commands should be wrapped by double quotes'
+        msg = """
+        INPUT ERROR: Second argument 'method' must be one of {}
```

Comment:
please include the numbers or variable names in the {} brackets.

This is a salt style guide thing.

Reply:
Fixed